content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -118,32 +118,104 @@ On your validator:
+
+    - Inbound traffic: Only from IP addresses of the stock `rippled` servers in the cluster you configured.
+
+    - Outbound traffic: Only to the IP addresses of the stock `rippled` servers in the cluster you configured and to <https://vl.ripple.com> through port 443.
+
+5. Restart `rippled`.
+
+        $ sudo systemctl restart rippled.service
+
+6. Use the [Peer Crawler](peer-crawler.html) endpoint on one of your stock `rippled` servers. The response should not include your validator. This verifies that your validator's `[peer_private]` configuration is working. One of the effects of enabling `[peer_private]` on your validator is that your validator's peers do not include it in their Peer Crawler results.
+
+<!-- { TODO: Future: add a recommended network architecture diagram to represent the proxy, clustering, and firewall setup: https://ripplelabs.atlassian.net/browse/DOC-2046 }-->
 
 
 ### Connect using public hubs
 
-This configuration involves connecting your validator to one or more public hubs that connect to the network. Successful public hubs embody the following traits:
+This configuration connects your validator to the network using two public hubs. This configuration is similar to [connecting using proxies you run yourself](#connect-using-proxies), but instead you connect through public hubs. Successful public hubs embody the following traits:
 
 - Good bandwidth.
 - Connections with a lot of reliable peers.
 - Ability to relay messages reliably.
 
-The benefit of connecting to public hubs is easy access to a lot of safe and reliable connections to the network. These connections help keep your validator healthy.
+**Pros:**
 
-To connect your validator to the network using public hubs, set the following configurations in your validator’s `rippled.cfg` file.
+  - Enables you to trust your validator's peers because you are connecting to public hubs.
+  - Provides easy access to safe connections to the network.
+  - Compared to [connecting using proxies](#connect-using-proxies), may be less likely to cause your validator to disconnect from the network due to a simultaneous peer outage.
 
-1. Include the following `[ips_fixed]` stanza. The two values, `r.ripple.com 51235` and `zaphod.alloy.ee 51235`, are public hubs. This stanza tells `rippled` to always attempt to maintain peer connections with these public hubs.
+**Cons:**
+
+  - Uses the default (most popular) public hubs, so it's distinctly possible that a public hub may be too busy to provide your validator with a connection to the network. Assuming that you can solely rely on a default public hub to provide your validator with a connection to the network may not be the best idea. If you want to rely on public hubs, it's better to have more of them. And, even better, use public hubs that aren't the default hubs whose duties include bootstrapping. ***TODO: where can users find other public hubs?***
```

Comment:
> ***TODO: where can users find other public hubs?***

There's no obvious answer for this at the moment. It's a matter of someone putting up a server with good specs, then building a strong reputation within the larger XRP Ledger community. It's possible that more hubs may become available, and if they do, you'd probably find out via the usual community hubs—XRPChat, Discord, Reddit, Twitter, etc.

Also, this paragraph combines the cons (if the hubs are too busy, you can't connect to the network) with the mitigations (use more hubs, including non-default ones). I recommend splitting those into separate paragraphs or bulleted lists.

>  And, even better, use public hubs that aren't the default hubs whose duties include bootstrapping. 

This sentence construction is slightly ambiguous regarding which hubs' duties include bootstrapping. Also, "bootstrapping" can be an arcane/obscure term, so it's better to use plainer language if possible. Suggested rephrase:

> Use non-default hubs, which are less likely to be busy helping new servers find other peers.